shogun-lib/src/main/java/de/terrestris/shogun/lib/model/security/permission/PublicEntity.java

```diff
@@ -0,0 +1,69 @@
+/* SHOGun, https://terrestris.github.io/shogun/
+ *
+ * Copyright © 2024-present terrestris GmbH & Co. KG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package de.terrestris.shogun.lib.model.security.permission;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.persistence.*;
+import lombok.*;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import java.io.Serializable;
+import java.time.OffsetDateTime;
+
+@Entity(name = "publicentities")
+@Table(schema = "shogun")
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@ToString(callSuper = true)
+@EqualsAndHashCode
+public class PublicEntity implements Serializable {
```

Comment:
As this class refers to permissions I would recommend to rename it to something like `PublicInstancePermission`. So it would be more in sync with the name of the `/permissions` controller and the existing permission models.